contracts/system-contracts/hedera-token-service/examples/hrc-904/HRC904Contract.sol

```diff
@@ -0,0 +1,107 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.7;
+
+import {IHRC719} from "../../IHRC719.sol";
+import {IHRC904TokenFacade} from "../../IHRC904TokenFacade.sol";
+import {IHRC904AccountFacade} from "../../../hedera-account-service/IHRC904AccountFacade.sol";
+
+// @title HRC904 Contract
+// @notice Provides interface for token airdrop, claim, reject and association operations
+// @dev Implements HRC-904 standard for token management operations
+//
+// Supports the following token operations:
+// - Cancelling pending airdrops for fungible and non-fungible tokens
+// - Claiming pending airdrops for fungible and non-fungible tokens  
+// - Rejecting tokens and NFTs
+// - Managing token associations and auto-association settings
+contract HRC904Contract {
+    event IsAssociated(bool status);
```

Comment:
where is this event used?

Reply:
it was leftover from earlier logic, I removed it